src/network.rs

```diff
@@ -0,0 +1,88 @@
+use errors::*;
+use std::path::{Path,PathBuf};
+use std::fs::{remove_file, rename};
+
+use args::Args;
+use fs;
+
+const BOOTPART_CONNECTIONS_DIR: &str = "system-connections";
```

Comment:
I think we need to move `BOOTPART_CONNECTIONS_DIR` as a full path "/mnt/boot/system-connections" in `args.rs`. That will go with a `_REDEFINE` variable name defined there as well for the integration tests. We need to do the same for `NM_CONNECTIONS_DIR` as well. Those two values can be added as `Args` members - `bootpart_connections_dir` and `nm_connections_dir`. Drawback is that this will make `args.rs` more crowded with command-specific items, but we can address this in a later PR.